Editor/CoreAPI/Models/DescribeFleetLocationResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿using AmazonGameLiftPlugin.Core.Shared;
+
+namespace AmazonGameLift.Editor
+{
+    public class DescribeFleetLocationResponse : Response
```

Comment:
See comment in fleet manager, consider renaming to `FindFleetLocationResponse`. IMO describe shouldn't be used here because it has a specific meaning in our service, but this is returning minimal info from that describe